### DISCUSSION TITLE
Slack or Discord

### DISCUSSION THREAD

**Discussion by skrawcz** [stitchfix/hamilton#112] [Polls]
Should we use Slack or Discord for the community chat server?

We currently use Discord (https://discord.gg/wCqxqBqn73) for the community.

**Comment by elijahbenizzy** [comment]
I've liked discord quite a bit but I voted for slack for two reasons:
1. _Join my discord server_ always feels spammy
2. Lower barrier to entry for slack than discord -- people using hamilton for their jobs are already there

**Comment by skrawcz** [comment]
For the record we chose slack - [link to join is here](https://join.slack.com/t/hamilton-opensource/shared_invite/zt-1bjs72asx-wcUTgH7q7QX1igiQ5bbdcg).
